config/system/const_variables.php

```diff
@@ -1,3 +1,4 @@
 <?php
 
 define('LIMIT_RECORD_PER_PAGE',  5);
+define('UPLOAD_MAX_FILE_SIZE',  10 * 1024 * 1024); // 10Mb
```

Comment:
Use the latest result, put the formula into comment

Reply:
fixed